packages/agent/src/agents/operators/documentAgent.ts

```
@@ -0,0 +1,155 @@
+    const k = config?.topK ?? this.topK;
+    const agentId = config?.agentId;
+    const results = await this.retrieveRelevantDocuments(query, k, agentId);
+
+    if (config?.raw) {
+      return results;
+    }
+
+    return this.formatDocumentsForContext(results);
+  }
+
+  public createDocumentChain(agentId: string): any {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)